tftrt/examples/cpp/image-classification/mnist.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef SAVED_MODEL_EXAMPLE_MNIST_H_
+#define SAVED_MODEL_EXAMPLE_MNIST_H_
+
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/lib/core/status.h"
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+namespace mnist {
+
+// MNISTImage contains the 28 x 28 pixel data for
+// an MNIST picture, in row major order.
+struct MNISTImage {
+  static const int kSize = 28;
+  float buf[kSize][kSize];
```

Comment:
Should we consider `constexpr` and `std::array` here as more canonical modern C++?